frontend/dockerfile/docs/reference.md

```diff
@@ -1116,6 +1116,32 @@ RUN apt-get update && apt-get install -y ...
 > The alternative syntax is supported for backward compatibility, but discouraged
 > for the reasons outlined above, and may be removed in a future release.
 
+### ENV inheritance in multi-stage builds
```

Comment:
I'm not sure such a long example is needed here - I think a simple statement that environment variables set with `ENV` are inherited from their parent stage, would be sufficient.

If we do have an example, I think it could be a lot shorter, and maybe have the variables that are set described in comments in the example instead of needing a separate table later?

Reply:
Agree with @jedevc. A short statement with basic example is enough. Could also cross link to https://docs.docker.com/build/building/multi-stage/ for further info on multi-stage builds.

Reply:
Fair enough. I suppose it's fairly similar to ENV inheritance from a base image.